doc/iterations/2023.6/scopes/scope Odesign converter.adoc

```diff
@@ -0,0 +1,16 @@
+= Scope
+
+== First task
+
+- [] Shape written, reviewed and merged
+
+== Second task
+
+- [] First prototype of the interfaces
+- [] First prototype of the implementation
+- [] Improvement of the first prototype after the discovery of performance issues
+- [] Second prototype of the backend merged
+
+== Third task
+
+- [ ] Integrated tests updated
```

Comment:
- Use a proper commit message, something like `[doc] Contribute the shape for the odesign to view converter` would be better. No need to reference the issue in your commit message since it would require you to add that to the CHANGELOG etc and you may even create multiple issues in the end for your task.
- Remove the scope file, it is intended to be maintained on your branch and be committed when the work is done. We will not merge this file beforehand and will not update it each time something is modified
- You can contribute all your shapes in one pull request, no need to create one pull request per task

Reply:
An use the new title of the commit message as the title of the pull request